src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,119 @@
 package core.basesyntax;
 
+import java.util.Map;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    static final int DEFAULT_INITIAL_CAPACITY = 16;
+    static final int ARRAY_SIZE_MULTIPLIER = 2;
+    static final int INDEX_FOR_NULL_KEY = 0;
+    private int size;
+    private int threshold;
+    private int capacity;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        capacity = DEFAULT_INITIAL_CAPACITY;
+        this.table = new Node[DEFAULT_INITIAL_CAPACITY];
+        this.threshold = (int) (DEFAULT_LOAD_FACTOR * capacity);
```

Comment:
You don't need key word "this" here

Reply:
agree, we don't need key word "this" here